client/dashboard/sites/settings-site-visibility/launch-form.tsx

```diff
@@ -1,12 +1,91 @@
+import { formatCurrency } from '@automattic/number-formatters';
+import { useQuery } from '@tanstack/react-query';
 import {
+	__experimentalHStack as HStack,
 	__experimentalText as Text,
 	__experimentalVStack as VStack,
 	Button,
+	ExternalLink,
 } from '@wordpress/components';
-import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, _n, sprintf } from '@wordpress/i18n';
 import { addQueryArgs } from '@wordpress/url';
+import { agencyBlogQuery } from '../../app/queries';
 import type { Site } from '../../data/types';
 
+function useAgencyBillingMessage( site: Site ) {
+	const { data, isLoading: isLoading, isError: isError } = useQuery( agencyBlogQuery( site.ID ) );
```

Comment:
```suggestion
	const { data, isLoading, isError } = useQuery( agencyBlogQuery( site.ID ) );
```

Reply:
Oops leftover code, good catch! Updated in bd4f56984ca95b314e913c8ad7f01524e6cf2eb7.